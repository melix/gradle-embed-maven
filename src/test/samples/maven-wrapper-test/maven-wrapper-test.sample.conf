executable: gradle
args: "clean --maven-args=-V"
expected-output-file: maven-wrapper-test.sample.out
allow-additional-output: true
