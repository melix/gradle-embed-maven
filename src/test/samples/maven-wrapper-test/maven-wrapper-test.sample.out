Apache Maven 3.5.4
